test/atm/global-workflow/jjob_ens_final.sh

```diff
@@ -47,9 +47,9 @@ fi
 
 # Execute j-job
 if [[ $machine = 'HERA' ]]; then
-    sbatch --ntasks=1 --account=$ACCOUNT --qos=batch --time=00:10:00 --export=ALL --wait ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_FINALIZE
+    sbatch --ntasks=1 --account=$ACCOUNT --qos=batch --time=00:10:00 --export=ALL --wait --output=atmensanlfinal-%j.out ${HOMEgfs}/jobs/JGLOBAL_ATMENS_ANALYSIS_FINALIZE
```

Comment:
The sbatch lines for hera, orion and hercules lines are identical.  I'm to blame for this.  We can add orion and hercules to the above machine test and then delete the elif for orion and hercules.  

I suspect this comment applies to all jjob*sh.

Reply:
@RussTreadon-NOAA Thanks for pointing that out. I saw it but wasn't sure if it was a placeholder for something in the future. I just committed some changes to remove the elif